spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -92,10 +92,9 @@ sealed class SpdxExpression {
     }
 
     /**
-     * Return all single licenses contained in the expression tree whereas a single license is one of:
-     * [SpdxLicenseIdExpression], [SpdxLicenseReferenceExpression] or [SpdxLicenseWithExceptionExpression].
+     * Return all single licenses contained in this expression as list of [SpdxSingleLicenseExpression]s.
```

Comment:
Commit message nits:
- "model" -> "reflect"?
- "single license with optional exception" -> "single license (with or without an exception)"?

Reply:
>     * "model" -> "reflect"?

Done.

>     * "single license with optional exception" -> "single license (with or without an exception)"?

Isn't that what optional means?

Reply:
Yes, somewhat, but in this context I found "optional" to sound a bit odd. But that's probably just me, let's keep it as-is.